contributors.txt

```diff
@@ -0,0 +1,57 @@
+GitExtensions Project Contributors Certification of Origin and Rights
+
+All contributors to GitExtensions must formally agree to abide by this
+certificate of origin by signing on the bottom with their github
+userid, full name, email address (you can obscure your e-mail, but it
+must be computable by human), and date.
+
+By signing this agreement, you are warranting and representing that
+you have the right to release code contributions or other content free
+of any obligations to third parties and are granting Henk Westhuis and
+GitExtensions project contributors, henceforth referred to as The GitExtensions
+Project, a license to incorporate it into The GitExtensions Project 
+or related works under the BSD license. You understand that The GitExtensions
+Project may or may not incorporate your contribution and you warrant 
+and represent the following:
+
+1. I am the creator of all my contributions. I am the author of all
+   contributed work submitted and further warrant and represent that
+   such work is my original creation and I have the right to license
+   it to The GitExtensions Project for release under the 3-clause BSD
```

Comment:
```suggestion
   it to The GitExtensions Project for release under the MIT and/or GPL
```

This will simplify to only state MIT once we complete the history review.